File: implementation/scipamato/core/entity/src/test/java/ch/difty/scipamato/core/entity/code/CodeTranslationTest.java
package ch.difty.scipamato.core.entity.code;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class CodeTranslationTest {

    @Test
    void comment() {
        CodeTranslation ntt = new CodeTranslation(1, "de", "code1", "comment", 1);
        assertThat(ntt.getComment()).isEqualTo("comment");
    }

    @Test
    void displayValue() {
        CodeTranslation ntt = new CodeTranslation(1, "de", "code1", "comment", 1);
        assertThat(ntt.getDisplayValue()).isEqualTo("de: code1");
    }

    @Test
    void field() {
        assertThat(CodeTranslation.CodeTranslationFields.COMMENT.getFieldName()).isEqualTo("comment");
    }
}

File: implementation/scipamato/core/entity/src/test/kotlin/ch/difty/scipamato/core/entity/code/CodeTranslationTest.kt
package ch.difty.scipamato.core.entity.code

import org.assertj.core.api.Assertions.assertThat

import org.junit.jupiter.api.Test

internal class CodeTranslationTest {

    @Test
    fun comment() {
        val ntt = CodeTranslation(1, "de", "code1", "comment", 1)
        assertThat(ntt.comment).isEqualTo("comment")
    }

    @Test
    fun displayValue() {
        val ntt = CodeTranslation(1, "de", "code1", "comment", 1)
        assertThat(ntt.displayValue).isEqualTo("de: code1")
    }

    @Test
    fun field() {
        assertThat(CodeTranslation.CodeTranslationFields.COMMENT.fieldName).isEqualTo("comment")
    }
}